config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "https://bluestonehomeloans.netlify.com/"
```

Comment:
Don't know if this should have been changed, but it seemed like it shoudl have..